javascript/README.md

```diff
@@ -0,0 +1,156 @@
+# PiranhaJS
+![Node.js CI](https://github.com/anag004/piranha/workflows/Node.js%20CI/badge.svg) ![Lint](https://github.com/anag004/piranha/workflows/Lint/badge.svg)
```

Comment:
These badges are for the wrong repo now.

Reply:
Done d840b4c6c66b9c0090b2c548971f8a2a93ac2e25